src/Streamly/Coreutils/Types.hs

```diff
@@ -0,0 +1,157 @@
+module Streamly.Coreutils.Types (
+      defaultCpOptions
+    , CpOptions (..)
+    , defaultUniqOptions
+    , UniqOptions (..)
+    , defaultCatOptions
+    , CatOptions (..)
+    , defaultEchoOptions
+    , EchoOptions (..)
+    , defaultHeadOptions
+    , HeadOptions
+    , defaultTailOptions
+    , TailOptions
+    , someFileToFP
+    , someDirToFP
+    , module Path
+    , module Path.Posix
+   )
+where
+
+import Path
+import Path.Posix
+      ( Path
+      , File
+      , Dir
+      , Abs
+      , Rel
+      , SomeBase (..)
+      , parseAbsFile
+      , parseRelFile
+      , parseSomeFile
+      , fromRelFile
+      , fromAbsFile
+      , fromRelFile
+      , parseAbsDir
+      , (</>)
+      , parseRelDir
+      , fromRelDir
+      , fromAbsDir)
+
+-------------------------------------------------------------------------------
+-- Record for options used with cp
+-------------------------------------------------------------------------------
+
+
+data CpOptions = CpOptions {
```

Comment:
You can move this into the `Cp` module itself. You want to put all `Cp` related code there.

Reply:
That way, you can use `verbose` as well instead of `cpVerbose`.

Reply:
:+1: I've changed the module structure.